internal/services/batch/batch_pool_resource.go

```diff
@@ -83,6 +84,19 @@ func resourceBatchPool() *pluginsdk.Resource {
 				MaxItems: 1,
 				Elem: &pluginsdk.Resource{
 					Schema: map[string]*pluginsdk.Schema{
+						// Property `node_deallocation_option` is set to be a writeOnly property by service team
+						// It can only perform on PUT operation and is not able to perform GET operation
+						// Here we treat `node_deallocation_option` the same as a secret value.
+						"node_deallocation_option": {
```

Comment:
option does not make any sense?
```suggestion
						"node_deallocation_methold": {
```

Reply:
Done